### ISSUE TITLE
Visual Grab and Release sticking around in two-balloon experiments

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#339]
Visual move cue (Space to Grab and Release) when two balloons are active is sticky. The visual cue does not disappear when the WASD keys and arrows appear. It usually only disappears once the balloon is release, but can stick around longer. 

It should disappear once grabbed and the WASD/Arrow keys cue shows up. This works fine in single balloon experiments.

<img width="902" alt="screen shot 2017-12-07 at 13 05 35" src="https://user-images.githubusercontent.com/1902116/33726366-640893b4-db4f-11e7-8b00-f6c1a8831f7c.png">

And sometimes sticks around loner. Here is a Tab release as an example.
<img width="905" alt="base-tab-release" src="https://user-images.githubusercontent.com/1902116/33726612-0dffe480-db50-11e7-9c7d-8f663d5a6064.png">

**Comment by jessegreenberg** [comment]
Thanks @terracoda! Ill look into it.

**Comment by jessegreenberg** [comment]
The blur event isn't happening on the green balloon, but it is on the  yellow balloon.

**Comment by jessegreenberg** [comment]
I guess it is, just not on the first time the green balloon is picked up.

**Comment by jessegreenberg** [comment]
Caused by moving the dragged balloon to the front.

**Comment by jessegreenberg** [comment]
@terracoda should be fixed in the above commit.
